packages/wonder-blocks-modal/components/modal-content.js

```diff
@@ -33,6 +33,10 @@ const styles = StyleSheet.create({
     wrapper: {
         flex: 1,
         overflow: "auto",
+
+        // This helps to ensure that the paddingBottom is preserved when
+        // the contents start to overflow, this goes away on display: flex
+        display: "block",
```

Comment:
This is such a non-obvious part of `flex-box`.  I don't understand why this decision was made in the spec.